CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+Thank you for your interest in contributing! Join the [**Plugin Marketplace channel**](https://community.mattermost.com/core/channels/plugins-marketplace) on the Mattermost community server for discussion about this plugin.
+
+## Reporting Issues
+
+If you think you found a bug, [please use the GitHub issue tracker](https://github.com/mattermost/mattermost-marketplace/labels/issues/new) to open an issue.
+
+## Community Plugin
+
+To add your plugin to the Marketplace, please open [an issue using thus template](https://github.com/mattermost/mattermost-marketplace/issues/new?template=add_plugin.md). To update your plugin, please also open an issue [using this template](https://github.com/mattermost/mattermost-marketplace/issues/new?template=update_plugin.md).
+
+### Playbook
+
+This is a playbook used my Mattermost core committers.
+
+#### New plugin
+
+JIRA template:
+  - [PM](https://mattermost.atlassian.net/browse/MM-22224)
+  - [Dev](https://mattermost.atlassian.net/browse/MM-22221)
+  - [Security](https://mattermost.atlassian.net/browse/MM-22225)
+  - [QA](https://mattermost.atlassian.net/browse/MM-22223)
+
+After a new plugin has been submitted, the assignees of the issue posts the following message:
+```
+## Process checklist
+- [ ] Request review and create JIRA tickets and link them here:
+    - PM:
+    - Dev:
+    - Security:
+    - QA:
+- [ ] Fork repo under Mattermost oranisation
```

Comment:
Did we want to ensure this repository is private to as to avoid polluting the "global namespace"? What should we do if there are naming conflicts (e.g. two `jira` plugins)? Should this fork occur before the JIRA tickets above, such that we do the review in our fork?

Reply:
> Did we want to ensure this repository is private to as to avoid polluting the "global namespace" ?

I've updated the wording to make clear it should be a private fork.

> What should we do if there are naming conflicts (e.g. two `jira` plugins)? 

Good question. I've updated the dev docs to make clear how a conflict should be resolved.


> Should this fork occur before the JIRA tickets above, such that we do the review in our fork?

I'm inclined to wait with the fork. A review might take a few week or month and iterations on the code will happen.